partials/functionCalls.vue

```diff
@@ -0,0 +1,269 @@
+<template>
+  <AppPanel class="function-calls">
+    <AppTable
+      v-for="(call, index) in functionCalls"
+      :key="index"
+      details
+    >
+      <AppTableBody>
+        <AppTableRow extend>
+          <AppTableCell
+            class="function-call-header"
+          >
+            <div>
+              <LabelType title="Function Call" />
+            </div>
+            <div class="accounts">
+              <Account
+                v-if="call.internalTx.accountId"
+                :value="call.internalTx.accountId"
+                title="Caller"
+                icon
+              />
+              <Account
+                v-if="call.internalTx.senderId"
+                :value="call.internalTx.senderId"
+                title="Sender"
+                icon
+              />
+              <Account
+                v-if="call.internalTx.recipientId"
+                :value="call.internalTx.recipientId"
+                title="Recipient"
+                icon
+              />
+              <Account
+                v-if="call.internalTx.oracleId"
+                :value="call.internalTx.oracleId"
+                title="Oracle"
+                icon
+              />
+            </div>
+          </AppTableCell>
+          <AppTableCell class="function-name">
+            <AppDefinition
+              title="Function"
+            >
+              {{ call.function }}
+            </AppDefinition>
+            <AppDefinition
+              v-if="call.internalTx.amount"
+              title="Amount"
+            >
+              <FormatAeUnit :value="call.internalTx.amount" />
+            </AppDefinition>
+            <AppDefinition
+              v-if="call.internalTx.queryFee"
+              title="Query Fee"
+            >
+              <FormatAeUnit :value="call.internalTx.queryFee" />
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.queryId"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Query ID"
+            >
+              {{ call.internalTx.queryId }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.query"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Query"
+            >
+              {{ call.internalTx.query }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.queryTtl"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Query Total"
```

Comment:
it means "time to live" (amount in blocks)

```suggestion
              title="Query TTL"
```